### DISCUSSION TITLE
Filament V3 How to custom the Page Resource navigation (icon, label, sort, group...)

### DISCUSSION THREAD

**Discussion by devhoussam1998** [Z3d0X/filament-fabricator#99] [Q&A] [answered]
I want to know How to custom the Page Resource navigation (icon, label, sort, group...)

**Comment by Z3d0X** [comment, marked as answer]
For labels you can publish the translation files & edit them there.
`php artisan vendor:publish --provider="\Z3d0X\FilamentFabricator\FilamentFabricatorServiceProvider"`

For Navigation Icon/Group/Sort, you can use the following in the boot method of a service provider
```php
// use Z3d0X\FilamentFabricator\Resources\PageResource;

PageResource::navigationIcon('heroicon-o-academic-cap');
PageResource::navigationGroup('Settings');
PageResource::navigationSort(1);
```
